src/main/java/ca/corefacility/bioinformatics/irida/ria/web/users/UsersAjaxController.java

```diff
@@ -24,112 +25,100 @@
-	 * @param userEditRequest a {@link UserEditRequest} containing details about a specific user
-	 * @param principal       a reference to the logged in user
-	 * @param request         the request
-	 * @return The name of the user view
-	 */
-	@RequestMapping(value = "/{userId}/edit", method = RequestMethod.POST)
-	public ResponseEntity<Map<String, String>> updateUser(@PathVariable Long userId,
-			@RequestBody UserEditRequest userEditRequest, Principal principal, HttpServletRequest request) {
-
-		UserDetailsResponse response = UIUsersService.updateUser(userId, userEditRequest, principal, request);
-
-		if (response.hasErrors())
-			return ResponseEntity.status(HttpStatus.BAD_REQUEST)
-					.body(response.getErrors());
-		else
-			return ResponseEntity.ok(null);
-
-	}
-
-	/**
-	 * Change the password for a user
-	 *
-	 * @param userId      The id of the user to edit (required)
-	 * @param oldPassword The old password of the user for password change
-	 * @param newPassword The new password of the user for password change
-	 * @param principal   a reference to the logged in user
-	 * @param request     the request
-	 * @return The name of the user view
-	 */
-	@RequestMapping(value = "/{userId}/changePassword", method = RequestMethod.POST)
-	public ResponseEntity<Map<String, String>> changeUserPassword(@PathVariable Long userId,
-			@RequestParam String oldPassword, @RequestParam String newPassword, Principal principal,
-			HttpServletRequest request) {
-
-		UserDetailsResponse response = UIUsersService.changeUserPassword(userId, oldPassword, newPassword, principal,
-				request);
-
-		if (response.hasErrors())
-			return ResponseEntity.status(HttpStatus.BAD_REQUEST)
-					.body(response.getErrors());
-		else
-			return ResponseEntity.ok(null);
-	}
-
-	/**
-	 * Get the projects associated with a user
-	 *
-	 * @param userId - the id for the user to show project details for
-	 * @return a {@link TableResponse} containing the list of projects associated with a users.
-	 */
-	@RequestMapping("/{userId}/projects/list")
-	public ResponseEntity<UserProjectDetailsResponse> getUserProjects(@PathVariable("userId") Long userId) {
-		return ResponseEntity.ok(UIUsersService.getUserProjects(userId));
-	}
-
-	/**
-	 * Get the details for a specific user
-	 *
-	 * @param userId      - the id for the user to show details for
-	 * @param mailFailure - if sending a user activation e-mail passed or failed
-	 * @param principal   - the currently logged in user
-	 * @return a {@link UserDetailsResponse} containing the details for a specific user
-	 */
-	@RequestMapping("/{userId}")
-	public ResponseEntity<UserDetailsResponse> getUserDetails(@PathVariable("userId") Long userId,
-			@RequestParam(value = "mailFailure", required = false, defaultValue = "false") final Boolean mailFailure,
-			Principal principal) {
-		return ResponseEntity.ok(UIUsersService.getUser(userId, mailFailure, principal));
-	}
-
+    private final UIUsersService UIUsersService;
```

Comment:
Can you fix the formatting of this file, it should be indented with tabs and not spaces. Might just need to update your ide to set tabs as default for .java files.

Reply:
Please see 242aeb0522f0fe6211596050c7f3a2dcc6ef7b32.